### DISCUSSION TITLE
Release notes improvement - Having to reorganize requests is very painful.

### DISCUSSION THREAD

**Discussion by JohnBarhorst-mastery** [Kong/insomnia#7328] [Feedback]
I have a lot of requests, spread across many folders. Release 9.0.0 is the second time in my short span of using Insomnia that all of my requests and folders have been resorted into a different order because of an update.
Reorganizing them is a slow and tedious process, even when the drag and drop is working correctly. 

If we could at least have a little blurb at the top of release notes that warns users of side effects like this in a release, that would be very helpful. More informative release notes in general would be pretty rad. I know it's a lot of work, but a list of PR titles is not really useful to folks.

**Comment by subnetmarco** [comment]
@JohnBarhorst-mastery we will look into this, we didn't want this to happen so we are digging into what caused the request order to change all of a sudden.

**Comment by JohnBarhorst-mastery** [comment]
@subnetmarco Thank you! The ordering seems flipped/reversed in this release. I'm noticing too that when I duplicate a request, it gets added above the request I'm duplicating instead of below, and adding new folders or requests is adding to the bottom of the list I'm adding to instead of the top.

I have manual sort enabled.

**Reply by subnetmarco** [reply]
Yes, thanks for letting us know. We identified an issue with reeuest sorting which will be fixed in 9.1 this week.

**Comment by subnetmarco** [comment]
This should have been fixed with the new 9.1 release.
